src/main/java/core/basesyntax/service/readandwrite/ReportWriterImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.readandwrite;
+
+import core.basesyntax.service.parser.CreateReport;
+import core.basesyntax.service.parser.Report;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ReportWriterImpl implements ReportWriter {
+    private static final String EXCEPTION_MESSAGE = "Can't write report in file ";
+    private Report report = new CreateReport();
+
+    @Override
+    public boolean writeReport(String pathToFile) {
```

Comment:
```suggestion
    public boolean writeReport(String pathToFile, String content) {
```
let's follow SRP and isoleta writing to file. In this case you will be able to reuse this service elsewhere

Reply:
done